src/request/plan.rs

```diff
@@ -234,3 +231,25 @@ impl<P: Plan<Result: HasLocks>, PdC: PdClient> Plan for ResolveLockPlan<P, PdC>
         }
     }
 }
+
+#[derive(Clone)]
+pub struct HeartbeatPlan<Req: TxnHeartBeatRequest, P: Plan> {
+    pub inner: P,
+    pub status: Arc<RwLock<TransactionStatus>>,
+}
+
+#[async_trait]
+impl<Req: TxnHeartBeatRequest, P: Plan> Plan for HeartbeatPlan<Req, P> {
+    type Result = Req::Response;
+
+    async fn execute(&self) -> Result<Self::Result> {
+        loop {
+            tokio::sleep(tokio::time::Duration::from_millis(5000));
```

Comment:
We should use a constant (or configurable interval) here instead of an integer literal. I suggest 10 seconds (TiDB's default).

Reply:
I have made it as a constant which is 10 seconds.

Reply:
I mean a `const` :)